requirements.txt

```diff
@@ -12,7 +12,6 @@ django-termsandconditions
 
 # django-cms
 django-cms
-cmsplugin-filer
```

Comment:
Do you know if this will affect any existing content created with the plugin? The description of the plugin is a bit vague, but at the same time sounds like it replaced some built in functionality. `A set of cms plugins that replace the plugins shipped with django-cms with versions that use file fields from django-filer.`

Reply:
djangocms-blog has this piece of information:

> Since cmsplugin-filer 1.1 has been moved to django-filer (since version 1.2).
https://djangocms-blog.readthedocs.io/en/latest/features/cmsplugin_filer.html#upgrading-django-cms-blog

And: https://djangocms-blog.readthedocs.io/en/latest/features/cmsplugin_filer.html#upgrading-djangocms-blog-from-0-9-x-to-1-0

So, I think this functionality is now built-in to django-filer and djangocms-picture/djangocms-file

Reply:
Perfect, this looks good then, thanks!